### DISCUSSION TITLE
Fuzzy completitions for surviving in a flatpak world

### DISCUSSION THREAD

**Discussion by Maykeye** [xonsh/xonsh#5202] [Q&A]
In fish  I can get `org.prismlauncher.PrismLauncher` from `pris<Tab><Tab>`. Does xonsh allows to autocomplete  such executables in a painless way? I see `FUZZY_PATH_COMPLETION` option, but seems to affect just paths -- not executables in $PATH.

**Comment by anki-code** [comment]
It's possible to implement as a feature like `FUZZY_PATH_COMPLETION`. 
Take a look into [path completer logic](https://github.com/xonsh/xonsh/blob/1674fdcc0b0534130f5e3b51ba8d49516ff71ef1/xonsh/completers/path.py#L283) and [commands completer](https://github.com/xonsh/xonsh/blob/1674fdcc0b0534130f5e3b51ba8d49516ff71ef1/xonsh/completers/commands.py).
PR is welcome!
